### DISCUSSION TITLE
MacFreeRDP and multimonitor settings. Is it possible?

### DISCUSSION THREAD

**Discussion by headintheclouds21** [FreeRDP/FreeRDP#8845] [Q&A] [answered]
Hi all,

First let me start off by saying I'm excited with MacFreeRDP and have enjoyed using it. Lately I've been trying to use MacFreeRDP with multiple monitors and I can't seem to figure out the settings to make it work. Has anyone had success with this? Does it even work?

I've tried various different settings to try and achieve this. Setting the use multimon to 1. I've tried change the screen mode id settings as well as the span monitors setting.

Does anyone know the correct settings to use in MacFreeRDP to accomplish multimonitor?

**Comment by akallabeth** [comment, marked as answer]
no. the mac client does not support it and `xfreerdp` on mac would require a `X11` server that is multi monitor capable.

**Reply by headintheclouds21** [reply]
That's unfortunate. It's helpful to know though. Thank you for the reply @akallabeth
